### PR TITLE
Simple Cartesian Planner

### PR DESCRIPTION
**Pull request by zkingston**
A simple Cartesian planner that uses the robowflex::Planner interface, as well as the Robot::IKQuery interface.

Computes paths by interpolating the EE pose and using IK to resolve joint configurations. Works well for, e.g., pre-grasping.
image

### CODE REVIEW

File: robowflex_library/include/robowflex_library/planning.h [hunk 1]
```diff
@@ -166,6 +167,72 @@ namespace robowflex
...
+    public:
+        /** \brief Constructor.
+         */
+        SimpleCartesianPlanner(const RobotPtr &robot, const std::string &name = "");
+
+        // non-copyable
+        SimpleCartesianPlanner(SimpleCartesianPlanner const &) = delete;
+        void operator=(SimpleCartesianPlanner const &) = delete;
+
+        /** \brief Plan a Cartesian motion (interpolation) given a \a request and a \a scene from a \a start
```
Comment (carlosquinterop) [inline robowflex_library/include/robowflex_library/planning.h:195]: There's no scene in this method

Comment (zkingston) [reply robowflex_library/include/robowflex_library/planning.h:195]: The scene is an optional field of the IKQuery here.

File: robowflex_library/scripts/ur5_cartesian.cpp [hunk 2]
```diff
@@ -0,0 +1,90 @@
+/* Author: Zachary Kingston */
+
+#include <robowflex_library/benchmarking.h>
```
Comment (carlosquinterop) [inline robowflex_library/scripts/ur5_cartesian.cpp:3]: This is not used here

File: robowflex_library/scripts/ur5_cartesian.cpp [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+#include <robowflex_library/builder.h>
+#include <robowflex_library/detail/ur5.h>
+#include <robowflex_library/geometry.h>
```
Comment (carlosquinterop) [inline robowflex_library/scripts/ur5_cartesian.cpp:6]: Not used?

**Review by carlosquinterop — APPROVED** [review]
LGTM. I tried the script and it works well on my end

**Comment by zkingston** [comment]
We need to do another clean-up run through the code sometime soon - run clang-tidy and what not and remove unnecessary includes everywhere else as well.

**Comment by ChamzasKonstantinos** [comment]
I think we should have a Fetch Example with this with the bookcase or a table environment. Fetch is high-Dof and could reveal some potential bugs

**Comment by zkingston** [comment]
There you go. Note that this is supposed to be a "simple" planner and won't be able to handle complex environments or manipulators in arbitrary scenarios - of course it will break in the scenarios you listed, and there are checks so that it returns failure in those cases.

**Pull request merged**